internal/services/compute/client/client.go

```diff
@@ -15,7 +15,6 @@ import (
 	"github.com/hashicorp/go-azure-sdk/resource-manager/compute/2021-11-01/dedicatedhostgroups"
 	"github.com/hashicorp/go-azure-sdk/resource-manager/compute/2021-11-01/dedicatedhosts"
 	"github.com/hashicorp/go-azure-sdk/resource-manager/compute/2021-11-01/sshpublickeys"
```

Comment:
Can we also update the other `2021-11-01` resources to use this updated API Version at the same time?

Reply:
Hi @tombuildsstuff, I have upgraded other `2021-11-01` resources and the test coverage of `compute` is the same as the `main` branch. Could you kindly have another review? Thanks.